quark/Objects/VarabileObject.py

```diff
@@ -21,72 +18,83 @@ def __init__(self, register_name, value, called_by_func=None):
         if called_by_func is not None:
             self._called_by_func.append(called_by_func)
 
+    def __repr__(self):
+        return "<VarabileObject-register:{}, value:{}, called_by_func:{}>".format(self._register_name, self._value,
+                                                                                  ','.join(self._called_by_func))
+
     @property
     def called_by_func(self):
         """
+        Record which functions have been called by using this register as a parameter.
 
-        :return:
+        :return: a list containing function name
         """
         return self._called_by_func
 
     @called_by_func.setter
     def called_by_func(self, called_by_func):
         """
+        Setter of called_by_func.
 
         :param called_by_func:
-        :return:
+        :return: None
         """
         self._called_by_func.append(called_by_func)
 
     @property
     def register_name(self):
         """
+        Individual register name, for example 'v3'.
 
-        :return:
+        :return: a string of register name
         """
         return self._register_name
 
     @register_name.setter
     def register_name(self, reg_name):
         """
+        Setter of register_name.
 
         :param reg_name:
-        :return:
+        :return: None
         """
         self._register_name = reg_name
 
     @property
     def value(self):
         """
+        The current value stored in the register.
 
-        :return:
+        :return: a string of the value
         """
         return self._value
 
     @value.setter
     def value(self, value):
         """
+        Setter of value.
 
         :param value:
-        :return:
+        :return: None
         """
         self._value = value
 
-    def get_all(self):
+    def get_current_status(self):
```

Comment:
The method is similar with __repr__. Is it necessary ?

Reply:
Same here, I will remove this function.